app/src/main/java/com/google/samples/apps/sunflower/GardenActivity.kt

```diff
@@ -88,4 +88,12 @@ class GardenActivity : AppCompatActivity() {
         // Pass any configuration change to the drawer toggle.
         drawerToggle.onConfigurationChanged(newConfig)
     }
+
+    override fun onBackPressed() {
+        with(binding.drawerLayout) {
+            val isDrawerOpen = isDrawerOpen(GravityCompat.START)
```

Comment:
let's inline `isDrawerOpen`

Reply:
I have updated.

Reply:
Thanks.  Sorry that I wasn't clear - no need for an extension function; let's just use:
```
if (isDrawerOpen(GravityCompat.START)) {
  closeDrawer...
...
```

Reply:
@tiembo done, check it.